README.md

```diff
@@ -0,0 +1,29 @@
+# Team Up Server API
+
+Simple node + express server implementing a REST API
+
+## Technologies
+
+- Node.js
+- Express
+- JWT Authentication
+
+### Version
```

Comment:
I would avoid this. It seems a good idea but this is one more obligation that you will have to change always the version changes. Also you would have to define versioning. This simple thing here carries a lot of side problems